src/shared/constants.ts

```diff
@@ -7,8 +7,6 @@ export const gitHubLicenseUrl = gitHubUrl + '/blob/master/LICENSE'
 // we can only convert this to an import when we find out how we can ignore
 // "'rootDir' is expected to contain all source files."
 // for this line
-const { version } = require('../../package.json')
-
-export const appVersion = version
+export const appVersion = '1.3.0'
```

Comment:
we should probably instruct our build script to update this value.

Reply:
Yup